docs/authoring/PartialArtifactsDownload.md

```diff
@@ -0,0 +1,26 @@
+# Supporting Partial Download of Artifacts
+
+Partial download of artifacts is now supported in release pipelines.
+This doc describes what the artifact extension author needs to do to get this feature.
+
+1. Provide a data source with the name "artifacts".
+2. Provide a data source with the name "artifactItems".
+
+### Role of the above 2 data sources in this feature.
+
+
+#### artifacts
+This data source is expected to return the base level artifact items that the corresponding artifact has published. For instance, in the below picture, the artifact _rogoyal-test has published 3 artifacts i.e drop1, drop2, drop3. These 3 artifacts are listed here are the result of the "artifacts" data source.
+
+![artifacts](./images/artifacts.png)
+
+Note: All the artifacts are of type "folder" and hence are expand-able. This takes us to the next data source.
```

Comment:
The artifacts can be of any type, the restriction is that this type needs to be mapped in browsableArtifactTypeMapping

Reply:
This is just for example. I have mentioned how to use that through browsable mapping in a later step. But I will make a change here to be clearer.